src/components/cluster/detail/ClusterApps.js

```diff
@@ -215,7 +215,7 @@ class ClusterApps extends React.Component {
 
   render() {
     return (
-      <React.Fragment>
+      <>
```

Comment:
Could be refactored to return early like

```
if (!this.props.showInstalledAppsBlock) {
  return null;
}
```

but this is fine I guess

Reply:
:+1: I'm not a big fan of the `&&` conditional rendering pattern but at one point I accepted it into the codebase as part of another review. This was way before the other front end engineers joined too. 

We can always discuss if we want to avoid that pattern, for now let's keep it.